content/get-started/installation/_index.md

```diff
@@ -5,60 +5,88 @@ slug: /get-started/installation
 
 # Install Streamlit
 
-There are multiple ways to set up your development environment and install Streamlit. Read below to
-understand these options. Developing locally with Python installed on your own computer is the most
+There are multiple ways to set up your development environment and install Streamlit.
+Developing locally with Python installed on your own computer is the most
 common scenario.
 
-## Summary for experts
+<Tip>
+
+Try a Streamlit Playground that runs in your browser &mdash; no installation required.
+(Note that this is not how Streamlit is meant to be used, because it has many downsides. That's why it's a
+_playground_!)
+
+<IconLink
+    href="/get-started/installation/streamlit-playground"
```

Comment:
Non-blocking: Why not just link straight to the playground? Isn't it self-explanatory enough?